build.sh

```diff
@@ -16,42 +16,44 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
-# Some things depend on HOME and it may not be set. We should fix those things, but until then, we just patch a value in
-if [ -z "$HOME" ]; then
-    export HOME=$DIR/artifacts/home
-
-    [ ! -d "$HOME" ] || rm -Rf $HOME
-    mkdir -p $HOME
-fi
-
-while [[ $# > 0 ]]; do
-    lowerI="$(echo $1 | awk '{print tolower($0)}')"
-    case $lowerI in
-        --docker)
-            export BUILD_IN_DOCKER=1
-            export DOCKER_IMAGENAME=$2
-            shift
-            ;;
-        *)
-            args+=( $1 )
-            ;;
-    esac
-    shift
-done
-
-# $args array may have empty elements in it.
-# The easiest way to remove them is to cast to string and back to array.
-temp="${args[@]}"
-args=($temp)
-
-dockerbuild()
-{
-    BUILD_COMMAND=/opt/code/build_projects/dotnet-host-build/build.sh $DIR/scripts/dockerrun.sh --non-interactive "$@"
```

Comment:
build_projects contains three folder, I think we can only delete one:
-> dotnet-host-build, i think nothing in there is used anymore except by perhaps sign.proj
-> CLiFramework and shared-build-targets-utils is a dependency on how tests are run right now. This can be fixed later
-> Update dependencies can still be used as an independent tool